build warning/error: hashtree.c:45:11: error: 'b' may be used uninitialized

```
hashtree-x86_64-unknown-linux-musl> hashtree.c: In function 'hashtree_detect':
hashtree-x86_64-unknown-linux-musl> hashtree.c:45:11: error: 'b' may be used uninitialized [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmaybe-uninitialized-Werror=maybe-uninitialized8;;]
hashtree-x86_64-unknown-linux-musl>    45 |     if (b & bit_SHA) {
hashtree-x86_64-unknown-linux-musl>       |           ^
hashtree-x86_64-unknown-linux-musl> hashtree.c:42:17: note: 'b' was declared here
hashtree-x86_64-unknown-linux-musl>    42 |     uint32_t a, b, c, d;
hashtree-x86_64-unknown-linux-musl>       |                 ^
hashtree-x86_64-unknown-linux-musl> hashtree.c:58:11: error: 'c' may be used uninitialized [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmaybe-uninitialized-Werror=maybe-uninitialized8;;]
hashtree-x86_64-unknown-linux-musl>    58 |     if (c & bit_AVX) {
hashtree-x86_64-unknown-linux-musl>       |           ^
hashtree-x86_64-unknown-linux-musl> hashtree.c:42:20: note: 'c' was declared here
hashtree-x86_64-unknown-linux-musl>    42 |     uint32_t a, b, c, d;
hashtree-x86_64-unknown-linux-musl>       |                    ^
hashtree-x86_64-unknown-linux-musl> cc1: all warnings being treated as errors
hashtree-x86_64-unknown-linux-musl> make[1]: *** [<builtin>: hashtree.o] Error 1
hashtree-x86_64-unknown-linux-musl> make[1]: Leaving directory '/build/source/src'
hashtree-x86_64-unknown-linux-musl> make: *** [Makefile:2: libhashtree] Error 2
error: builder for '/nix/store/f9d8y8arcxa3dhds15j3s9ddxdn6c6fr-hashtree-x86_64-unknown-linux-musl.drv' failed with exit code 2;
       last 10 log lines:
       > hashtree.c:58:11: error: 'c' may be used uninitialized [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmaybe-uninitialized-Werror=maybe-uninitialized8;;]
       >    58 |     if (c & bit_AVX) {
       >       |           ^
       > hashtree.c:42:20: note: 'c' was declared here
       >    42 |     uint32_t a, b, c, d;
       >       |                    ^
       > cc1: all warnings being treated as errors
       > make[1]: *** [<builtin>: hashtree.o] Error 1
       > make[1]: Leaving directory '/build/source/src'
       > make: *** [Makefile:2: libhashtree] Error 2

```

--- a/src/hashtree.c
+++ b/src/hashtree.c
@@ -39,7 +39,7 @@ static hashtree_hash_fcn hash_ptr = init_and_hash;
 
 static hashtree_hash_fcn hashtree_detect() {
 #ifdef __x86_64__
-    uint32_t a, b, c, d;
+    uint32_t a = 0, b = 0, c = 0, d = 0;
     __get_cpuid_count(7, 0, &a, &b, &c, &d);
 
     if (b & bit_SHA) {

